roles/confluent.variables/defaults/main.yml

```diff
@@ -1101,9 +1106,372 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
+kafka_connect_replicator_consumer_ssl_cert_path: ""
+
+### Set to the location of your TLS key when configuring TLS for Kafka Connect Replicator Consumer.
+kafka_connect_replicator_consumer_ssl_key_path: ""
+
+### Set to the password of your TLS key when configuring TLS for Kafka Connect Replicator Consumer.
+kafka_connect_replicator_consumer_ssl_key_password: ""
+
+### Set to the location of your TLS TrustStore when configuring TLS using Keystores and TrustStores for Kafka Connect Replicator Consumer.
+kafka_connect_replicator_consumer_ssl_truststore_file_path: ""
+
+### Set to the location of your TLS KeyStore when configuring TLS using Keystores and TrustStores for Kafka Connect Replicator Consumer.
+kafka_connect_replicator_consumer_ssl_keystore_file_path: ""
+
+### Boolean that determines if Kerberos is enabled for the Kafka Connect Replicator Consumer configuration.
+kafka_connect_replicator_consumer_kerberos_enabled: "{{ true if kafka_connect_replicator_consumer_kerberos_principal is defined else false }}"
+
+### Boolean that determines if SASL SCRAM is enabled for the Kafka Connect Replicator Consumer configuration.
+kafka_connect_replicator_consumer_sasl_scram_enabled: "{{ true if kafka_connect_replicator_consumer_sasl_scram_username is defined else false }}"
+
+### SCRAM password for the Consumer to authenticate with.
+kafka_connect_replicator_consumer_sasl_scram_password: "{{ sasl_scram_users.kafka_connect_replicator.password }}"
+
+### Boolean that determines if SASL PLAIN is enabled for the Kafka Connect Replicator Consumer configuration.
+kafka_connect_replicator_consumer_sasl_plain_enabled: "{{ true if kafka_connect_replicator_consumer_sasl_plain_username is defined else false }}"
+
+### SASL PLAIN password for the Consumer to authenticate with.
+kafka_connect_replicator_consumer_sasl_plain_password: "{{ sasl_plain_users.kafka_connect_replicator.password }}"
+
+kafka_connect_replicator_consumer_security_protocol: "{% if kafka_connect_replicator_consumer_ssl_enabled==true and (kafka_connect_replicator_consumer_kerberos_enabled==true or kafka_connect_replicator_consumer_sasl_scram_enabled==true or kafka_connect_replicator_consumer_sasl_plain_enabled==true) %}SASL_SSL{% elif kafka_connect_replicator_consumer_ssl_enabled==false and (kafka_connect_replicator_consumer_kerberos_enabled==true or kafka_connect_replicator_consumer_sasl_scram_enabled==true or kafka_connect_replicator_consumer_sasl_plain_enabled==true) %}SASL_PLAINTEXT{% elif kafka_connect_replicator_consumer_ssl_enabled==true and (kafka_connect_replicator_consumer_kerberos_enabled==false or kafka_connect_replicator_consumer_sasl_scram_enabled==false or kafka_connect_replicator_consumer_sasl_plain_enabled==false) %}SSL{% else %}NONE{% endif %}"
+
+kafka_connect_replicator_consumer_truststore_path: "/var/ssl/private/kafka_connect_replicator_consumer.truststore.jks"
```

Comment:
this is the first component we are allowing diff keystore/truststore sets on. it has been requested that we do this in kafka but i've pushed back. im wondering if its a good idea to do it here

Reply:
Agreed.  Because this could be different CA's for different clusters, I was trying to break it all out.  Trying to cover as wide of expanse of config as possible.